README.md

```diff
@@ -54,6 +54,16 @@ Build for Spark 3
 ./dev/make-distribution.sh -Pspark-3 -Plog4j-1/-Plog4j-2
 `
 
+Build for Spark 2 on MacOS Apple M1/Arm(For testing purposes only, not recommended for production.)  
+`
+./dev/make-distribution.sh -Pspark-2 -Plog4j-1/-Plog4j-2 -Dos.detected.name=osx -Dos.detected.arch=x86_64 -Dos.detected.classifier=osx-x86_64 -Dscala.binary.version=2.11
```

Comment:
I guess it's unnecessary to specify -Dscala.binary.version?

Reply:
It is necessary to specify -Dscala.binary.version, otherwise the build will fail

![image](https://user-images.githubusercontent.com/8604206/178729862-c53ac383-a333-4a92-ba20-6bbbd41bd7aa.png)
![image](https://user-images.githubusercontent.com/8604206/178729914-81acec78-c501-4714-b101-80ca2fda34e1.png)

Reply:
> It is necessary to specify -Dscala.binary.version, otherwise the build will fail
> 
> ![image](https://user-images.githubusercontent.com/8604206/178729862-c53ac383-a333-4a92-ba20-6bbbd41bd7aa.png) ![image](https://user-images.githubusercontent.com/8604206/178729914-81acec78-c501-4714-b101-80ca2fda34e1.png)

I found the reason for the build failure, when maven version=3.8.5, the build fails without specifying -Dscala.binary.version. When maven version=3.6.3, the build does not need to specify -Dscala.binary.version. I have removed the -Dscala.binary.version option.

![image](https://user-images.githubusercontent.com/8604206/178740089-a8b84fee-8763-4d92-8c9c-0cdbcdb4cb12.png)

![image](https://user-images.githubusercontent.com/8604206/178739977-2c17fb9d-8479-41bf-b29d-41cad3433de9.png)

@waitinfuture

Reply:
It's strange since -Pspark-2 and -Pspark-3 specifies scala version. We will check it tomorrow because I don't have a M1 book...